app/components/shared/inputs/ValidatedForm.vue.ts

```diff
@@ -52,4 +52,8 @@ export default class ValidatedForm extends Vue {
   emitInput(data: any, event: Event) {
     this.$emit('input', data, event);
   }
+
+  get apiErrors() {
+    return this.$validator.errors.first('api');
+  }
```

Comment:
Not sure it's the best way to handle errors since `ValidatedForm.vue.ts` is an abstraction that shouldn't know about API. Also, this field doesn't use any validations. You actually use it to just show a error-message. I think it's better to choose one of the next solutions:
1) Handle such errors that are not related to any field as somethink like `global` or `wildcard` errors instead `api-error`
2) Handle such errors directly in WidgetEditor or  Widget template as we did before. There is  no need for `nextTick` call in this way.